langfuse/src/openai/traceMethod.ts

```diff
@@ -26,14 +26,21 @@ const wrapMethod = async <T extends GenericMethod>(
     modelParameters,
     name: config?.generationName,
     startTime: new Date(),
+    promptName: config?.langfusePrompt?.name,
+    promptVersion: config?.langfusePrompt?.version,
   };
 
   let langfuseParent: LangfuseParent;
   const hasUserProvidedParent = config && "parent" in config;
 
   if (hasUserProvidedParent) {
     langfuseParent = config.parent;
-    observationData = { ...config, ...observationData };
+    observationData = {
+      ...config,
+      ...observationData,
+      promptName: config?.promptName ?? config?.langfusePrompt?.name, // Maintain backward compatibility for users who use promptName
```

Comment:
can we simplify by not having this here and in observationData, is one place enough (only here?)